src/config.ts

```diff
@@ -43,18 +48,22 @@ export interface Config {
   relayerCsc: {
     isEnabled: boolean;
   }
+  reverseRelayerCsc:{
```

Comment:
let's add a comment on both `relayerCsc` and `reverseRelayerCsc` to indicate what exact those two means

Reply:
added